app/src/components/loginWindow.ts

```diff
@@ -18,13 +18,15 @@ export async function createLoginWindow(
     resizable: false,
     webPreferences: {
       nodeIntegration: true, // TODO work around this; insecure
+      contextIsolation: false,
     },
   });
   await loginWindow.loadURL(
     `file://${path.join(__dirname, 'static/login.html')}`,
   );
 
-  ipcMain.once('login-message', (event, usernameAndPassword) => {
+  ipcMain.on('login-message', (event, usernameAndPassword) => {
```

Comment:
Same here: pls comment and link to https://github.com/electron/electron/issues/28017